tests/e2e/testfiles/values.yaml

```diff
@@ -420,57 +449,87 @@
   modules:
-    - "observability"
+    - "authorizationproxyserver"
     - "authorization"
+    - "observability"
   steps:
     - "Given an environment with k8s or openshift, and CSM operator installed"
+    - "Create [authorization-proxy-server] prerequisites from CR [1]"
     - "Apply custom resource [1]"
-    - "Validate custom resource [1]"
-    - "Validate [powerscale] driver from CR [1] is installed"
-    - "Validate [authorization] module from CR [1] is installed"
-    - "Validate [observability] module from CR [1] is installed"
-    - "Disable [observability] module from CR [1]"
-    - "Validate [powerscale] driver from CR [1] is installed"
-    - "Validate [authorization] module from CR [1] is installed"
-    - "Validate [observability] module from CR [1] is not installed"
-    - "Disable [authorization] module from CR [1]"
-    - "Set secret for driver from CR [1] to [isilon-creds]"
-    - "Validate [powerscale] driver from CR [1] is installed"
-    - "Validate [authorization] module from CR [1] is not installed"
-    - "Validate [observability] module from CR [1] is not installed"
+    - "Validate [authorization-proxy-server] module from CR [1] is installed"
+    - "Configure authorization-proxy-server for [powerscale]"
+    - "Create storageclass with name [op-e2e-isilon] and template [testfiles/powerscale-templates/powerscale-storageclass-template.yaml] for [pscale]"
+    - "Set up secret with template [testfiles/powerscale-templates/powerscale-auth-secret-template.yaml] name [isilon-creds-auth] in namespace [isilon] for [pscaleAuth]"
+    - "Set up secret with template [testfiles/powerscale-templates/karavi-authorization-config.json] name [karavi-authorization-config] in namespace [isilon] for [pscaleAuthSidecar]"
+    - "Set up secret with template [testfiles/authorization-templates/rootCertificate.pem] name [proxy-server-root-certificate] in namespace [isilon] for [authSidecarCert]"
+    - "Apply custom resource [2]"
+    - "Validate custom resource [2]"
+    - "Validate [powerscale] driver from CR [2] is installed"
+    - "Validate [authorization] module from CR [2] is installed"
+    - "Validate [observability] module from CR [2] is installed"
+    - "Disable [observability] module from CR [2]"
+    - "Validate [powerscale] driver from CR [2] is installed"
+    - "Validate [authorization] module from CR [2] is installed"
+    - "Validate [observability] module from CR [2] is not installed"
+    - "Disable [authorization] module from CR [2]"
+    - "Set secret for driver from CR [2] to [isilon-creds]"
+    - "Validate [powerscale] driver from CR [2] is installed"
+    - "Validate [authorization] module from CR [2] is not installed"
+    - "Validate [observability] module from CR [2] is not installed"
+    - "Run custom test"
     # cleanup
-    - "Enable forceRemoveDriver on CR [1]"
-    - "Delete custom resource [1]"
+    - "Enable forceRemoveDriver on CR [2]"
+    - "Delete custom resource [2]"
```

Comment:
Missing clean up steps to delete authorization server and restoring templates

Reply:
Added cleanup.